jwst/datamodels/util.py

```diff
@@ -57,16 +57,26 @@ def open(init=None, memmap=False, **kwargs):
         - dict: The object model tree for the data model
 
     memmap : bool
-        Turn memmap of FITS file on or off.  (default: False).  Ignored for
-        ASDF files.
+        Turn memmap of file on or off.  (default: False).
 
     kwargs : dict
-        Additional keyword arguments passed to lower level functions. These arguments
-        are generally file format-specific. Arguments of note are:
+        Additional keyword arguments passed to the DataModel constructor.  Some arguments
+        are general, others are file format-specific.  Arguments of note are:
+
+        - General
+
+           validate_arrays : bool
+             If `True`, arrays will be validated against ndim, max_ndim, and datatype
+             validators in the schemas.
 
         - FITS
 
-           skip_fits_update - bool or None
+           cast_fits_arrays : bool
+             If `True`, arrays will be cast to the dtype described by the schema
+             when read from a FITS file.
+             If `False`, arrays will be read without casting.
```

Comment:
This is a new parameter available in stdatamodels 0.3.  If I'm using an earlier one, will it just ignore it?  Or do we need to specify `stdatamodels>=0.3` in setup.cfg for this PR?

Reply:
Yes excellent point!  We'll need the requirement I think.